src/components/Home.js

```diff
@@ -51,7 +60,8 @@ const Home = () => {
   return (
     <>
       {bookList}
-      <h1>Book list empty</h1>
+      <div className="bar-horiz" />
+      <h1 className="book-state">Book list empty</h1>
       <AddBook />
     </>
```

Comment:
- [ ] @Matthieu96Code kindly check on your form, it's going beyond the background, it should have a space left below the inputs, kindly check the comparison below for a better understanding

Zeplin Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/230712236-1cdf7ca7-8983-49a9-845f-43c9df8cc5d2.png) |  ![image](https://user-images.githubusercontent.com/61408860/230712255-8a5efeb3-3787-4630-b8db-3332f15d2a96.png)